src/Common/src/Interop/Windows/mincore/Interop.GetCommPorts.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.CoreComm_L1_1_2, SetLastError = true)]
+        unsafe private static extern int GetCommPorts(
+             uint* lpPortNumbers,
+             uint uPortNumbersCount,
+             out uint puPortNumbersFound);
+
+        unsafe internal static int GetCommPorts(
```

Comment:
Nit: per https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md, visibility should come first.  We then generally put `unsafe` before the return type, e.g. `internal static unsafe int`.